react/features/dynamic-branding/functions.js

```diff
@@ -13,3 +13,24 @@ export function extractFqnFromPath(path: string) {
 
     return parts.length > 2 ? `${parts[len - 2]}/${parts[len - 1]}` : '';
 }
+
+/**
+ * Returns the url used for fetching dynamic branding.
+ *
+ * @param {Object} state - The state of the app.
+ * @returns {string}
+ */
+export function getDynamicBrandingUrl(state: Object) {
+    const { dynamicBrandingUrl } = state['features/base/config'];
+
+    if (dynamicBrandingUrl) {
+        return dynamicBrandingUrl;
+    }
+
+    const baseUrl = state['features/base/config'].brandingDataUrl;
```

Comment:
So, is this a backwards compatibility thing? If so, can you please add a TODO comment so we remove it eventually?

Reply:
It's more of a branding specific thing...

On the branding side (where we have `brandingDataUrl` set), we must have the tenant otherwise we don't know for which client we're requesting the data.